Modules/SubModule_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -130,11 +130,16 @@ function xASL_wrp_ResampleASL(x)
 %% ------------------------------------------------------------------------------------------
 % 6. Create mean control image, if available, in native & standard space
 if  nVolumes>1    
-    % Create mean control in native space
-    ControlIm = xASL_quant_GetControlLabelOrder(xASL_io_Nifti2Im(x.P.Path_rdespiked_ASL4D));
-    IM_mean = xASL_stat_MeanNan(ControlIm, 4);
-    xASL_io_SaveNifti(x.P.Path_rdespiked_ASL4D, x.P.Path_mean_control, IM_mean, [], 0);
-
+    if x.modules.asl.bTimeEncoded % ===== Here we should just configure the mean control (first volume)==== %

```

Comment:
Is the first image always a control image for time encoded ASL? For creating the mean control image, are there not multiple control images that we could average here?

Reply:
For @amahroo 's data this seems to be the case?

Reply:
For Hadamard case, the first image is considered to be the only control image (because all the sub-boli are in control state), so that's the only control image